library/build.gradle

```diff
@@ -26,33 +26,44 @@ jar {
     }
 }
 
+wrapper {
+    gradleVersion = '7.3.1'
+}
+
 dependencies {
+    def fuelVersion = '2.3.1'
+    def log4jVersion = '2.16.0'
+
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.2"
 
-    //core
-    implementation 'com.github.kittinunf.fuel:fuel:2.2.1'
+    // https://mvnrepository.com/artifact/com.github.kittinunf.fuel/fuel
+    implementation "com.github.kittinunf.fuel:fuel:${fuelVersion}"
+    implementation "com.github.kittinunf.fuel:fuel-coroutines:${fuelVersion}"
+
+    // https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-slf4j-impl
+    implementation "org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}"
 
-    implementation 'org.slf4j:slf4j-log4j12:1.7.5'
-    implementation 'io.github.microutils:kotlin-logging:1.7.7'
+    // https://mvnrepository.com/artifact/io.github.microutils/kotlin-logging
+    implementation 'io.github.microutils:kotlin-logging:2.1.16'
 
     implementation 'com.rabbitmq:amqp-client:5.8.0'
 
     // https://mvnrepository.com/artifact/com.google.inject/guice
-    compile group: 'com.google.inject', name: 'guice', version: '4.2.2'
-
-    //packages
-    implementation 'com.github.kittinunf.fuel:fuel-coroutines:2.2.1'
+    implementation 'com.google.inject:guice:5.0.1'
 
     // https://mvnrepository.com/artifact/io.reactivex.rxjava2/rxkotlin
     implementation("io.reactivex.rxjava2:rxkotlin:2.4.0")
 
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    testImplementation group: 'junit', name: 'junit', version: '4.12'
+
+    // https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-test
+    testImplementation 'org.jetbrains.kotlin:kotlin-test:1.6.0'
 }
 
 compileKotlin {
     kotlinOptions.jvmTarget = "1.8"
```

Comment:
Btw! I've seen that there are some references to Java 8-only APIs (removed in Java 11) &mdash; the entire `java.xml.bind` package, e.g. in `com.oddin.oddsfeedsdk.api.ApiClient`. Would it make sense to mention that users of the library in Java 9+ should apply some workaround to avoid linkage errors in the runtime? as discussed on https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception/43574427#43574427